styles/sass/pages/_contact-us.scss

```diff
@@ -57,13 +57,18 @@ a{
 
   &__form{
     display: block;
+    margin: 0 auto;
     padding-top: 30px;
     text-align: left;
+    width: 304px;
+
+    &__container:nth-child(1),&__container:nth-child(2) {
```

Comment:
add a space after the `,` like this
` &__container:nth-child(1), &__container:nth-child(2) {`